guide/07-working-with-bigdata/06-summarizing-data.ipynb

```diff
@@ -82,7 +82,7 @@
     "      <td><a href=\"https://enterprise.arcgis.com/en/portal/latest/use/geoanalytics-summarize-within.htm\"><p align=\"left\">summarize_within</p></a></td>\n",
```

Comment:
<p>All the text in this guide below the Note displays as <strong>bold. </strong>Do you see that on your side when previewing locally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Perfect!